src/components/Banner/tests/Banner.test.tsx

```diff
@@ -134,4 +135,31 @@ describe('<Banner />', () => {
       expect(div.getDOMNode()).toBe(document.activeElement);
     });
   });
+
+  describe('context', () => {
+    it('passes the within banner context', () => {
+      const Child: React.SFC<{}> = (_props) => {
+        return (
+          <BannerContext.Consumer>
+            {(BannerContext) => (BannerContext ? <div /> : null)}
+          </BannerContext.Consumer>
+        );
+      };
+
+      const banner = mountWithAppProvider(
+        <BannerContext.Provider value>
```

Comment:
We can remove this, with it we're not testing that `Banner` provides the context.